transports/quic/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 ## 0.9.3 - unreleased
 
+- Close quinn's `Endpoint` with `Ok(())` when `Accept` returns `None`. See [PR 4621].
```

Comment:
The changelog is written for users, not for maintainers. I think this is a good git commit message. I think for a changelog you could write something along the lines of:

> No longer report error when explicit closing of a QUIC endpoint succeeds.

Reply:
```suggestion
- No longer report error when explicit closing of a QUIC endpoint succeeds.
  See [PR 4621].
```

Reply:
yeah makes sense, thanks guys!